future_challenges_for_RSEng.tex

```diff
@@ -69,6 +69,43 @@ \section{Aspects of this future}
 
 \subsection{What do we want to have achieved by 2035}
 
+Looking ahead over a 10 year timeframe, we hope that the RSE community will have
+supported a number of changes and developments that help to make research software outputs
+more sustainable, robust and maintainable. We see this happening in a number of ways. However,
+we want to ensure that one of the core achievements made by the RSE community in this time
+frame is putting the infrastructure and training opportunities in place, alongside the necessary
+advocacy, to ensure that researchers receive core software skills as part of their scientific training.
+With development of software tools or scripts becoming an increasingly important part of everyday
+research activities, this is vital and it will allow RSEs to focus on other more advanced and specialist
+areas. In this context, we do not foresee the demise of RSE as an independent role. However, we
+do see the work of RSEs developing and shifting as the research community changes.
+
+The specialist skills that RSEs provide take a significant amount of time to develop and maintain.
+We would, therefore, not expect domain researchers to also become experts in a range of
+advanced RSE topics. 
+At the same time, we do recognise that the application of core software development best practices
+to enhance and address problems with research codebases, that can often make up the bulk of
+current RSE work, will change. As noted above, these are skills that we
+would expect any researcher who writes code to have in 10 years time. 
+The rapid development of AI means that researchers are likely to have access to tooling that can assist
+them in writing robust, sustainable code, lowering the barrier to developing code, or at least changing the
+required skill set to some extent.
+They can also be expected to have an understanding of frameworks to support testing, packaging and
+deployment of their code. Again, all these skills will, we anticipate, be gained as part of training
+infrastructure that will provide domain scientists with software development expertise as part of their
```

Comment:
When we are talking about infrastructure here, we can also extend the discussion to integration with RDM infrastuctures.